dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/SchedulerController.java

```diff
@@ -94,7 +94,8 @@
     @ApiOperation(value = "createSchedule", notes = "CREATE_SCHEDULE_NOTES")
     @ApiImplicitParams({
             @ApiImplicitParam(name = "processDefinitionId", value = "PROCESS_DEFINITION_ID", required = true, dataType = "Int", example = "100"),
-            @ApiImplicitParam(name = "schedule", value = "SCHEDULE", dataType = "String", example = "{'startTime':'2019-06-10 00:00:00','endTime':'2019-06-13 00:00:00','crontab':'0 0 3/6 * * ? *'}"),
+            @ApiImplicitParam(name = "schedule", value = "SCHEDULE", dataType = "String",
+                    example = "{'startTime':'2019-06-10 00:00:00','endTime':'2019-06-13 00:00:00', 'timezoneId':'', 'crontab':'0 0 3/6 * * ? *'}"),
```

Comment:
Keep space style consistent in `example`

Reply:
Done